packages/lesswrong/components/tagging/TagSubforumPage.tsx

```diff
@@ -92,6 +117,26 @@ export const TagSubforumPage = ({ classes }: { classes: ClassesType}) => {
   const sortBy = query.sortBy || subforumDefaultSorting;
 
   const { tag, loading, error } = useTagBySlug(slug, "TagSubforumFragment");
+  
+  const { openDialog } = useDialog();
+  
+  const { results: members, totalCount: membersCount } = useMulti({
+    terms: {view: 'tagCommunityMembers', profileTagId: tag?._id},
```

Comment:
```suggestion
    terms: {view: 'tagCommunityMembers', profileTagId: tag?._id, limit: 0},
```
Or making this a resolver only field on the tag. It just feels a bit wasteful to fetch all the user info and then only show the count. Although empirically it doesn't seem to make much difference to the load time atm so you don't need to do this